runtime/Cargo.toml

```diff
@@ -64,6 +65,7 @@ runtime-benchmarks = [
     'frame-system/runtime-benchmarks',
     'pallet-balances/runtime-benchmarks',
     'pallet-timestamp/runtime-benchmarks',
+    'pallet-vesting/runtime-benchmarks',
```

Comment:
Just for note:
It isn't required. It needed if you runs benchmarking for runtime and wants __including__ results for that pallet.